seedfarmer/commands/_parameter_commands.py

```diff
@@ -36,6 +36,13 @@ def generate_export_env_params(metadata: Optional[Dict[str, Any]]) -> Optional[L
     return envs
 
 
+def generate_export_raw_env_params(metadata: Optional[Dict[str, Any]]) -> Optional[List[str]]:
+    envs: List[str] = []
+    if metadata is not None and metadata.keys() is not None:
+        envs = [f"export {upper_snake_case(k)}={metadata[k]}" for k in metadata.keys()]
```

Comment:
this appears to result in an export statement like:
```
export SOME_VARIABLE=some_value
```
but if the value is JSON (which is common) or the value has spaces in it, then the export won't work correctly. should probably wrap the value in single quotes like:
```
export SOME_VARIABLE='some value`
```

Reply:
Added and corrected